packages/material-ui-lab/src/SliderStyled/SliderStyled.d.ts

```diff
@@ -48,4 +48,46 @@ export const SliderValueLabel: React.FC<SliderValueLabel>;
  */
 declare const Slider: ExtendSliderUnstyled<SliderStyledTypeMap>;
 
+export type SliderStyledClassKey =
+  /** Styles applied to the root element. */
+  | 'root'
```

Comment:
This won't do anything. JSDOC for literals is ignored: https://github.com/microsoft/TypeScript/issues/25499.

Reply:
Aha, now I understand why I didn't have these on the node directly, so I had to iterate all `leadingComments` and see which one is right before the specific `classKey` - see https://github.com/mui-org/material-ui/pull/23370#discussion_r516142560 I am open to other ideas of how to do this.